cartography/intel/azure/compute.py

```diff
@@ -0,0 +1,227 @@
+import logging
+from typing import Dict
+from typing import List
+
+import neo4j
+from azure.mgmt.compute import ComputeManagementClient
+
+from .credentials import Credentials
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+def get_client(credentials: Credentials, subscription_id: str) -> ComputeManagementClient:
+    client = ComputeManagementClient(credentials, subscription_id)
+    return client
+
+
+def get_vm_list(credentials: Credentials, subscription_id: str) -> List[Dict]:
+    try:
+        client = get_client(credentials, subscription_id)
+        vm_list = list(map(lambda x: x.as_dict(), client.virtual_machines.list_all()))
+
+        for vm in vm_list:
+            x = vm['id'].split('/')
+            vm['resource_group'] = x[x.index('resourceGroups') + 1]
+
+        return vm_list
+
+    except Exception as e:
+        logger.warning(f"Error while retrieving virtual machines - {e}")
+        return []
+
+
+def load_vms(neo4j_session: neo4j.Session, subscription_id: str, vm_list: List[Dict], update_tag: int) -> None:
+    ingest_vm = """
+    UNWIND {vms} AS vm
+    MERGE (v:VirtualMachine{id: vm.id})
```

Comment:
[question] What do you think about making this `AzureVirtualMachine`, or making this have 2 labels: `VirtualMachine` (like a generic label regardless of the clouds we support) _and_ `AzureVirtualMachine`.

Reply:
I like AzureVirtualMachine more than VirtualMachine :). I have renamed `VirtualMachine` to `AzureVirtualMachine`. I will create an issue for so that we can create a generic Label for VirtualMachine across clouds.